### PR TITLE
Add missing packages error reporting to Minify and Less tasks

### PR DESCRIPTION
**Pull request by MAXakaWIZARD**
If required package is not installed, detailed detailed error message will be shown:
```bash
Class lessc not found. Please install leafo/lessphp Composer package
```

### CODE REVIEW

File: src/Task/Assets/Less.php [hunk 1]
```diff
@@ -141,6 +145,10 @@ protected function lessphp($file)
      */
     protected function less($file)
     {
+        if (!class_exists('\Less_Parser')) {
+            return Result::error($this, 'Less_Parser not found. Please install oyejorge/less.php composer package');
```
Comment (DavertMik) [inline src/Task/Assets/Less.php]: why not `Result::errorMissingPackage` then?

File: src/Task/Assets/Less.php [hunk 2]
```diff
@@ -128,6 +128,10 @@ protected function compile($file)
      */
     protected function lessphp($file)
     {
+        if (!class_exists('\lessc')) {
+            return Result::error($this, 'lessc not found. Please install leafo/lessphp composer package');
```
Comment (DavertMik) [inline src/Task/Assets/Less.php]: why not Result::errorMissingPackage then?

Comment (MAXakaWIZARD) [inline src/Task/Assets/Less.php]: Ah, sorry, my bad. Forgot to add it there (it was there in my test project)

**Comment by MAXakaWIZARD** [comment]
@DavertMik fixed

**Comment by DavertMik** [comment]
Thanks!

**Pull request merged**